docs/features/ems_reporting.md

```diff
@@ -41,24 +42,24 @@ A scheduled Lambda task will run nightly that collects distribution events and b
 
 Information about requesting EMS account can be found on EMS website.  Here are basic steps in order to submit reports to EMS.
 
-1. Get a provider account on the EMS file server and obtain access to their UAT or OPS environment.
+### 1) Get a provider account on the EMS file server and obtain access to their UAT or OPS environment
 
 Provide IP addresses, data provider name , contact information (primary and secondary) to EMS, and EMS will set up account and firewall rules to allow applications to send files to EMS.
 For Cumulus instances running on NGAP, the IP address should be the Elastic IP (IPv4 Public IP field) of the `NGAP NAT Instance` in EC2, and that should be the IP that EMS firewall sees for any instance in that account.
 
-2. Request updates on NGAP NACL.
+### 2) Request updates on NGAP NACL
 
 For Cumulus instances running on NGAP, submit a [NGAP service desk ticket](https://bugs.earthdata.nasa.gov/servicedesk/customer/portals), and specify "Exception / Enhancement” request for “Network / Whitelist” changes to the account, that will add EMS host IP to the NACL (Network Access Control List) to allow outbound traffic from NGAP Application VPCs to EMS host.
 
-3. Send public key to EMS. Lambda will provide private key when sftp files to EMS.
+### 3) Send public key to EMS. Lambda will provide private key when sftp files to EMS
 
 Upload the corresponding private key to s3, use `{{system_bucket}}` as bucket name and `{{prefix}}/crypto/ems-private.pem` as key,  `{{system_bucket}}` and `{{prefix}}` are configured in `app/config.yml`.  If a different private key file name other than `ems-private.pem` is used, specify it in the `ems` configuration in `app/config.yml`.
```

Comment:
Needs to be updated to remove references to `app/config.yml`? Or is that going to happen separately? I also want to remove the reference to variables showing handlebar syntax (`{{system_bucket}}`), since that isn't relevant for TF anymore

Reply:
Yeah, good catch on the app/config.yml.   This needs sorted.

Reply:
I'm debating if we want to make this configurable or not, as we do not have it set-up as a passthrough configuration to the archive module.  🤔